src/sentry/runner/commands/devserver.py

```diff
@@ -17,10 +17,11 @@
 @click.option('--reload/--no-reload', default=True, help='Autoreloading of python files.')
 @click.option('--watchers/--no-watchers', default=True, help='Watch static files and recompile on changes.')
 @click.option('--workers/--no-workers', default=False, help='Run asynchronous workers.')
+@click.option('--refresh/--no-refresh', default=False, help='Automatic browser refreshing on webpack builds')
```

Comment:
I'd prefer almost anything more clear than `--refresh`. That word doesn't mean much to me. I personally don't see the issue with `--livereload`. I don't really think that's weird with another flag named `--reload` imo. But I don't  have a strong opinion. `--browser-reload` seems fine to me.